docs/templating/twig_function_reference/cart_twig_functions.md

```diff
@@ -0,0 +1,24 @@
+---
+description: Cart Twig functions enable checking whether product can be added to cart and formatting the price.
+---
+
+# Cart Twig functions
+
+### `ibexa_can_be_added_to_cart`
+
+The `ibexa_can_be_added_to_cart()` checks whether the provided product can be added to cart by eliminating base products (products that have variants), products that are not available, products that do not have a price that corresponds to a currency selected for the cart, and products, for which VAT category is not set.
```

Comment:
> eliminating base products (products that have variants)

It should be phrased better to make it clear that you can't add Base Product to cart because you need to choose specific variant of that product.